### DISCUSSION TITLE
How do I Open a Port On Repl?

### DISCUSSION THREAD

**Discussion by sacrosaunt** [GiorgioBrux/nitro-sniper-enhanced#65] [Q&A] [answered]
Hello, how can I open a port on repl to work with UptimeRobot and which port do I use?

**Comment by DarkGD123** [comment]
Read the instructions in the main page @Imploded, also i wont recommend replit because you get rate limited pretty quick

**Comment by GiorgioBrux** [comment, marked as answer]
The port is 80, you just need to set `replit` to true and it will automatically open a server on the 80 port.
